tests/integration/600.framework-detection.test.js

```diff
@@ -293,16 +293,15 @@ test('should pass framework-info env to framework sub process', async (t) => {
     await builder
       .withPackageJson({
         packageJson: {
-          dependencies: { gatsby: '^3.0.0' },
-          scripts: { develop: 'node -p process.env.GATSBY_LOGGER' },
+          dependencies: { nuxt3: '^2.0.0' },
```

Comment:
I've changed this because the Gatsby plugin (which will now run as part of Netlify Dev) needs Node 14, whereas CLI needs to support Node 12. I've changed it to Nuxt 3, which doesn't have a plugin.